HMD/Assets/Scenes/scripts/ConnectionHandler.cs

```diff
@@ -2,20 +2,23 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public class test : MonoBehaviour
+public class ConnectionHandler : MonoBehaviour
 {
-    public TSScConnection TSSc;
-
     // Start is called before the first frame update
+    public TSScConnection TSSc;
     void Start()
     {
-        TSSc = new TSScConnection();
-        TSSc.ConnectToHost("172.24.196.222", 0);
+        TSSc.ConnectToHost("172.24.196.222", 2, 14141);
```

Comment:
indexing starts from 0 on the TSS Server, so Team Cartographer is Team 1, please update the 2 to a 1